CHANGES.rst

```diff
@@ -59,6 +59,9 @@ Unreleased
    reddit.
  * **[REDDIT]** Added ``DeprecationWarning`` to :meth:`login` as reddit will
    stop supporting cookie-based authentication on 2015/08/03.
+ * **[FEATURE]** Added :meth:`hide` and :meth:`unhide`, which accept up to 50
```

Comment:
Please add the feature to the end of the feature list section. I like to try to keep them grouped by category.